operations/validate.go

```diff
@@ -60,17 +73,29 @@ func Validate() cli.Command {
 				}
 				catcher := grip.NewSimpleCatcher()
 				for _, file := range files {
-					catcher.Add(validateFile(filepath.Join(path, file.Name()), ac, quiet, long))
+					catcher.Add(validateFile(filepath.Join(path, file.Name()), ac, quiet, long, localModuleMap))
 				}
 				return catcher.Resolve()
 			}
 
-			return validateFile(path, ac, quiet, long)
+			return validateFile(path, ac, quiet, long, localModuleMap)
 		},
 	}
 }
+func getLocalModulesFromInput(localModulePaths []string) (*map[string]string, error) {
+	moduleMap := make(map[string]string)
+	catcher := grip.NewBasicCatcher()
+	for _, module := range localModulePaths {
+		pair := strings.Split(module, "=")
+		if len(pair) < 2 {
+			catcher.Add(errors.Errorf("problem parsing local module '%s'", module))
```

Comment:
catcher.Errorf (and maybe explicitly say why it's an invalid format)

Reply:
done